.github/install/deploywithsampleapp.yaml

```diff
@@ -0,0 +1,15 @@
+name: istio
+services:
+  istioinstallation:
+    type: $(#use https://raw.githubusercontent.com/Revolyssup/service-mesh-patterns/import/samples/istioinstall.yaml)
+    settings: 
+        controlplaneNS: istio-system
+        namespace: default
+        istioVersion: 1.12.0
```

Comment:
Can this grab the latest Istio release version from GitHub.com/istio/istio and use the latest version?

Reply:
The logic to do that is in the workflow. version will be overridden to latest during execution